pkg/scheduler/pool_sizing.go

```diff
@@ -66,7 +67,7 @@ func (s *WorkerPoolSizer) Start() {
 			// Handle case where we want to make sure all available manually provisioned nodes have available workers
 			if s.workerPoolConfig.Mode == types.PoolModeExternal {
 				err := s.occupyAvailableMachines()
-				if err != nil {
+				if err != nil && !strings.Contains(err.Error(), "redislock: not obtained") {
```

Comment:
Approved, but I think you could also check the error type from redislock which might be cleaner instead of hardcoding the string

Reply:
Gonna use that instead.

Reply:
👍